doc/source/contributing.rst

```diff
@@ -207,6 +206,29 @@ See the full conda docs `here <http://conda.pydata.org/docs>`__.
 
 At this point you can easily do an *in-place* install, as detailed in the next section.
 
+.. _contributing.windows:
+
+Creating a Windows development environment
+------------------------------------------
+
+To build on windows, you need to have compilers installed to build the extensions. You will need to install the appropriate Visual Studio compilers, VS 2008 for Python 2.7, VS 2010 for 3.4, and VS 2015 for Python 3.5.
+
+For Python 2.7, you can install the ``mingw`` compiler which will work equivalently to VS2008::
+
+      conda install -n pandas_dev libpython
+
+or use the [Microsoft Visual Studio VC++ compiler for Python](https://www.microsoft.com/en-us/download/details.aspx?id=44266). Note that you have to check the ``x64`` box to install the ``x64`` extension building capability.
+
+For Python 3.4, you can download and install the [Windows 7.1 SDK](https://download.microsoft.com/download/A/6/A/A6AC035D-DA3F-4F0C-ADA4-37C8E5D34E3D/winsdk_web.exe)
+For Python 3.5, you can download and install the [Visual Studio 2015 Community Edition](https://www.visualstudio.com/en-us/downloads/visual-studio-2015-downloads-vs.aspx).
+
+Here are some references:
+
+- https://github.com/conda/conda-recipes/wiki/Building-from-Source-on-Windows-32-bit-and-64-bit
+- https://cowboyprogrammer.org/building-python-wheels-for-windows/
+- https://blog.ionelmc.ro/2014/12/21/compiling-python-extensions-on-windows/
+- https://support.enthought.com/hc/en-us/articles/204469260-Building-Python-extensions-with-Canopy
+
```

Comment:
@gfyoung yes, we basically recommend using a conda install now as its the most straightforward / easiest way of doing this (on windows). my impression was that ``mingw-64`` just became available / stable. I'd like to see this in common use before we recommend its use to build.

In any event, simply create another issue and can discuss there.